Makefile

```diff
@@ -5,22 +5,27 @@ SPARQL=sparql
 ROBOT_ENV=ROBOT_JAVA_ARGS=-Xmx12G
 ROBOT=$(ROBOT_ENV) robot
 
-all: $(BUILD_DIR)/phenoscape-ontology-classified.ofn
+all: $(BUILD_DIR)/phenoscape-kb.ttl $(BUILD_DIR)/phenoscape-kb-tbox-hierarchy.ofn $(BUILD_DIR)/qualities.txt $(BUILD_DIR)/anatomical_entities.txt
 
 clean:
 	rm -rf $(BUILD_DIR)
 
+# Create build directory
+$(BUILD_DIR): clean
+	mkdir -p $@
+
 # Ontologies.ofn - list of ontologies to be imported
 # Mirror ontologies locally
 $(BUILD_DIR)/mirror: ontologies.ofn
```

Comment:
This should depend on $(BUILD_DIR) I think.

Reply:
Yeah, thanks.